test/com/facebook/buck/go/testdata/go_test/subtests.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+// modified from an example here: https://blog.golang.org/subtests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestTime(t *testing.T) {
```

Comment:
What would happen if Buck runs this tests with go that is lower than 1.7?

Reply:
It will most likely break. Should I skip that test for now and put a comment to enable it only after Buck officially supports Go 1.7?

Reply:
The proper solution would be to detect the version of go and enable this test only when it's higher than 1.7. Check `GoAssumptions.assumeGoCompilerAvailable();` that is called in the test to make sure go is available. You'll need to extend it to detect the version of go.

Reply:
Added the check for Go version. Please take another look